.github/workflows/debug_appimagegenerator-prod(dev).yml

```diff
@@ -0,0 +1,57 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app to Azure Web App - appimagegenerator-prod
+
+on:
+  push:
+    branches:
+      - debug
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up .NET Core
+        uses: actions/setup-dotnet@v1

```

Comment:
Need to use `@v3` versions of all `actions/` here and below

Reply:
@amrutha95 can you look at it?

Reply:
Hey @khmyznikov, as Michael said it looks like v3 is the latest version. We should update this on our other workflows as well. Any reason not to?